public/app/plugins/datasource/loki/queryUtils.ts

```diff
@@ -108,6 +108,17 @@ export function getNormalizedLokiQuery(query: LokiQuery): LokiQuery {
   return { ...rest, queryType: LokiQueryType.Range };
 }
 
+export function parseToArray(query: string): string[] {
```

Comment:
```suggestion
export function parseToNodeNamesArray(query: string): string[] {
```

I am wondering if we should be more specific with name here, because array could be anything.